demo/mobile/config.xml

```diff
@@ -1,93 +1,71 @@
-    <!-- Spotlight Icon -->
-    <icon src="www/res/icons/ios/Icon-40.png" width="40" height="40" />
-    <icon src="www/res/icons/ios/Icon-40@2x.png" width="80" height="80" />
-    <!-- iPhone / iPod Touch -->
-    <icon src="www/res/icons/ios/icon.png" width="57" height="57" />
-    <icon src="www/res/icons/ios/icon@2x.png" width="114" height="114" />
-    <!-- iPad -->
-    <icon src="www/res/icons/ios/Icon-72.png" width="72" height="72" />
-    <icon src="www/res/icons/ios/Icon-72@2x.png" width="144" height="144" />
-    <!-- iPhone Spotlight and Settings Icon -->
-    <icon src="www/res/icons/ios/Icon-small.png" width="29" height="29" />
-    <icon src="www/res/icons/ios/Icon-small@2x.png" width="58" height="58" />
-    <!-- iPad Spotlight and Settings Icon -->
-    <icon src="www/res/icons/ios/Icon-50.png" width="50" height="50" />
-    <icon src="www/res/icons/ios/Icon-50@2x.png" width="100" height="100" />
-  </platform>
-    <plugin name="cordova-plugin-device" spec="~1.1.2" />
+    <content src="index.html" />
+    <access origin="*" />
+    <access origin="https://fonts.googleapis.com" />
+    <access origin="https://maps.googleapis.com" />
+    <access origin="https://*.gstatic.com" />
+    <access origin="https://*.amazonaws.com" />
+    <preference name="webviewbounce" value="false" />
+    <preference name="UIWebViewBounce" value="false" />
+    <preference name="DisallowOverscroll" value="true" />
+    <preference name="android-minSdkVersion" value="16" />
+    <preference name="BackupWebStorage" value="none" />
+    <feature name="StatusBar">
+        <param name="ios-package" onload="true" value="CDVStatusBar" />
+    </feature>
+    <platform name="android">
+        <icon density="ldpi" src="www/res/icons/android/icon.png" />
+        <icon density="mdpi" src="www/res/icons/android/icon-mdpi.png" />
+        <icon density="hdpi" src="www/res/icons/android/icon-hdpi.png" />
+        <icon density="xhdpi" src="www/res/icons/android/icon-xhdpi.png" />
+        <icon density="xxhdpi" src="www/res/icons/android/icon-xxhdpi.png" />
+        <icon density="xxxhdpi" src="www/res/icons/android/icon-xxxhdpi.png" />
+        <splash density="land-hdpi" src="www/res/splash/android/splash-land-hdpi.png" />
+        <splash density="land-ldpi" src="www/res/splash/android/splash-land-ldpi.png" />
+        <splash density="land-mdpi" src="www/res/splash/android/splash-land-mdpi.png" />
+        <splash density="land-xhdpi" src="www/res/splash/android/splash-land-xhdpi.png" />
+        <splash density="port-hdpi" src="www/res/splash/android/splash-port-hdpi.png" />
+        <splash density="port-ldpi" src="www/res/splash/android/splash-port-ldpi.png" />
+        <splash density="port-mdpi" src="www/res/splash/android/splash-port-mdpi.png" />
+        <splash density="port-xhdpi" src="www/res/splash/android/splash-port-xhdpi.png" />
+        <preference name="SplashScreen" value="screen" />
+    </platform>
+    <platform name="ios">
```

Comment:
I don't think I understand, before the PR the config.xml already had platform tags.
If you mean that if icons and splash tags should be outside the platform tags, they work inside and outside, but I think it's better organised if you put them inside.